cmd/acr/purge_test.go

```diff
@@ -606,6 +650,68 @@ var (
 	invalidLastUpdateTime = "date"
 
 	OneTagResult = &acr.RepositoryTagsType{
+		Response: autorest.Response{
+			Response: &http.Response{
+				StatusCode: 200,
+			},
+		},
+		Registry:  &testLoginURL,
+		ImageName: &testRepo,
+		TagsAttributes: &[]acr.TagAttributesBase{
+			{
+				Name:                 &tagName,
+				LastUpdateTime:       &lastUpdateTime,
+				ChangeableAttributes: &acr.ChangeableAttributes{DeleteEnabled: &deleteEnabled},
+				Digest:               &digest,
+			},
+		},
+	}
+
+	OneTagResultWithNext = &acr.RepositoryTagsType{
+		Response: autorest.Response{
+			Response: &http.Response{
+				StatusCode: 200,
+				Header:     http.Header{linkHeader: {"/acr/v1/&testRepo/_tags?last=latest&n=3&orderby=timedesc rel=\"next\""}},
```

Comment:
Shouldn't we test the format `</acr/v1/&testRepo/_tags?last=latest&n=3&orderby=timedesc>; rel="next"` ?

Reply:
@adusumillipraveen Any ideas on this? and other similar ones.

Reply:
I changed it and it breaks one of my test , but it works on actual repos i am trying to purge. I might be missing something in there, will spend some time to see if anything else is trimming them in case of non stubbed code.

Reply:
I'm wondering if there is parsing problem when the link is like `</acr/v1/&testRepo/_tags?n=3&orderby=timedesc&</acr/v1/&testRepo/_tags?last=latest&n=3&orderby=timedesc>; rel="next">; rel="next"`

Reply:
Yes, I am not sure of that scenario is really need though. But i have pushed a fix to address that. Please let me know if that looks ok.